later/lib/database/models/record_model.dart

```diff
@@ -4,10 +4,21 @@ part 'record_model.g.dart';
 
 @collection
 class RecordModel {
+  RecordModel({
+    this.title,
+    this.description,
+    required this.url,
+    required this.lastEditedAt,
+  }) : createdAt = DateTime.now();
+
+  final String url;
+  final String? description;
+  final DateTime createdAt;
+  final DateTime lastEditedAt;
```

Comment:
These fields must not be private as per [isar's documentation](https://pub.dev/packages/isar#quickstart) (the DB library we use)